### PR TITLE
Add FluidMechanics project

### PR DESCRIPTION
**Pull request by DonsetPG**
Hi,

This was an idea from @araffin, tell me if you need any more informations for the documentation. 

Thanks for this awesome project! 

Have a nice day,
Paul

### CODE REVIEW

**Comment by araffin** [comment]
Please update the changelog too, otherwise LGTM ;)

File: docs/misc/projects.rst [hunk 1]
```diff
@@ -122,3 +122,17 @@ Reinforcement learning is used to train agents to control pistons attached to a
 | Email: jackberkowitz88@gmail.com
 | GitHub: https://github.com/jaberkow/WaveRL
 | Presentation: http://bit.ly/WaveRLslides
+
+Fenics-DRL: Fluid mechanics and Deep Reinforcement Learning
+-----------------------------------------------------------
+
+Deep Reinforcement Learning is used to train agents to control the position or the shape of obstacles in different fluids. `Fenics <https://fenicsproject.org>`_ is used for the Fluid Mechanics part, and Stable Baselines is used for the DRL.
```
Comment (araffin) [inline docs/misc/projects.rst]: Maybe add the reason why you want to control position and shape of obstacles.
I think you can also symplify "train agents to control" to "to control"

File: docs/misc/projects.rst [hunk 2]
```diff
@@ -122,3 +122,17 @@ Reinforcement learning is used to train agents to control pistons attached to a
 | Email: jackberkowitz88@gmail.com
 | GitHub: https://github.com/jaberkow/WaveRL
 | Presentation: http://bit.ly/WaveRLslides
+
+Fenics-DRL: Fluid mechanics and Deep Reinforcement Learning
+-----------------------------------------------------------
+
+Deep Reinforcement Learning is used to train agents to control the position or the shape of obstacles in different fluids. `Fenics <https://fenicsproject.org>`_ is used for the Fluid Mechanics part, and Stable Baselines is used for the DRL. 
+We hope to keep using it but with other Fluid Mechanics libraries.
```
Comment (araffin) [inline docs/misc/projects.rst]: You can remove that part ;)

**Comment by DonsetPG** [comment]
Should be good now I guess :)

**Pull request merged**